lib/taiko.js

```diff
@@ -1621,19 +1621,58 @@ module.exports.highlight = highlight;
 
 async function highlight(selector, ...args) {
   validate();
+  if (!defaultConfig.highlightOnAction) {
+    throw new Error('Highlights are disabled. Please enable highlights.');
+  }
+  let elem = (await $$xpath('//*'))[0];
+  await evaluate(elem, addTaikoHighlightStyleIfNotPresent);
 
   function highlightNode() {
+    let _class = 'taiko_highlight_style';
     if (this.nodeType === Node.TEXT_NODE) {
-      this.parentElement.style.outline = '0.5em solid red';
-      return;
+      return this.parentElement.classList.add(_class);
     }
-    this.style.outline = '0.5em solid red';
+    this.classList.add(_class);
   }
   let elems = await handleRelativeSearch(await findElements(selector), args);
   await evaluate(elems[0], highlightNode);
   descEvent.emit('success', 'Highlighted the ' + description(elems[0], true));
 }
 
+function addTaikoHighlightStyleIfNotPresent() {
+  let taikoHighlightStyleID = 'taiko_highlight';
+  if (document.getElementById(taikoHighlightStyleID)) {
+    return;
+  }
+  let head = document.head || document.getElementsByTagName('head')[0];
+  let style = document.createElement('style');
+  let css = '.taiko_highlight_style { outline: 0.5em solid red; }';
+  head.appendChild(style);
+  style.type = 'text/css';
+  style.id = taikoHighlightStyleID;
+  style.appendChild(document.createTextNode(css));
+}
+
+/**
+ * Clear all highlights for the current page.
```

Comment:
change to:

`Clear all highlights marked using @highlight on the current page.`